app/src/main/java/fr/free/nrw/commons/explore/map/ExploreMapFragment.java

```diff
@@ -598,60 +568,60 @@ public void displayBottomSheetWithInfo(final Marker marker) {
+        binding.bottomSheetDetailsBinding.commonsButton.setVisibility(place.hasCommonsLink() ? View.VISIBLE : View.GONE);
+        binding.bottomSheetDetailsBinding.commonsButton.setOnClickListener(
             view -> Utils.handleWebUrl(getContext(), place.siteLinks.getCommonsLink()));
 
         int index = 0;
         for (Media media : mediaList) {
             if (media.getFilename().equals(place.name)) {
                 int finalIndex = index;
-                mediaDetailsButton.setOnClickListener(view -> {
+                binding.bottomSheetDetailsBinding.mediaDetailsButton.setOnClickListener(view -> {
                     ((ExploreMapRootFragment) getParentFragment()).onMediaClicked(finalIndex);
                 });
             }
             index++;
         }
-        title.setText(place.name.substring(5, place.name.lastIndexOf(".")));
-        distance.setText(place.distance);
+        binding.bottomSheetDetailsBinding.title.setText(place.name.substring(5, place.name.lastIndexOf(".")));
+        binding.bottomSheetDetailsBinding.category.setText(place.distance);
         // Remove label since it is double information
         String descriptionText = place.getLongDescription()
             .replace(place.getName() + " (", "");
         descriptionText = (descriptionText.equals(place.getLongDescription()) ? descriptionText
             : descriptionText.replaceFirst(".$", ""));
         // Set the short description after we remove place name from long description
-        description.setText(descriptionText);
+        binding.bottomSheetDetailsBinding.description.setText(descriptionText);
     }
 
     @Override
     public void addSearchThisAreaButtonAction() {
-        searchThisAreaButton.setOnClickListener(presenter.onSearchThisAreaClicked());
+        binding.searchThisAreaButton.setOnClickListener(presenter.onSearchThisAreaClicked());
     }
 
     @Override
     public void setSearchThisAreaButtonVisibility(boolean isVisible) {
         if (isVisible) {
-            searchThisAreaButton.setVisibility(View.VISIBLE);
+            binding.searchThisAreaButton.setVisibility(View.VISIBLE);
         } else {
-            searchThisAreaButton.setVisibility(View.GONE);
+            binding.searchThisAreaButton.setVisibility(View.GONE);
         }
     }
 
     @Override
     public void setProgressBarVisibility(boolean isVisible) {
         if (isVisible) {
-            progressBar.setVisibility(View.VISIBLE);
+            binding.mapProgressBar.setVisibility(View.VISIBLE);
         } else {
-            progressBar.setVisibility(View.GONE);
+            binding.mapProgressBar.setVisibility(View.GONE);
         }
     }
 
     @Override
     public boolean isDetailsBottomSheetVisible() {
-        if (bottomSheetDetails.getVisibility() == View.VISIBLE) {
+        if (binding.bottomSheetDetailsBinding.getRoot().getVisibility() == View.VISIBLE) {
             return true;
         } else {
             return false;
```

Comment:
You are testing a _boolean expression_ and then returning a boolean result.  This whole method could be reduced to 
```java
public boolean isDetailsBottomSheetVisible() {
    return binding.bottomSheetDetailsBinding.getRoot().getVisibility() == View.VISIBLE;
}
```
Similarly for the other methods that test for visibility.